packages/openapi-v3/src/enhancers/extensions/security.spec.extension.ts

```diff
@@ -6,37 +6,36 @@
 import {bind} from '@loopback/core';
 import debugModule from 'debug';
 import {inspect} from 'util';
-import {
-  mergeOpenAPISpec,
-  ReferenceObject,
-  SecuritySchemeObject,
-} from '../../../..';
-import {asSpecEnhancer, OASEnhancer} from '../../../../enhancers/types';
-import {OpenApiSpec} from '../../../../types';
+import {mergeOpenAPISpec, ReferenceObject, SecuritySchemeObject} from '../..';
+import {OpenApiSpec} from '../../types';
+import {asSpecEnhancer, OASEnhancer} from '../types';
 const debug = debugModule('loopback:openapi:spec-enhancer');
 
 export type SecuritySchemeObjects = {
   [securityScheme: string]: SecuritySchemeObject | ReferenceObject;
 };
 
 export const SECURITY_SCHEME_SPEC: SecuritySchemeObjects = {
-  bearerAuth: {
+  jwt: {
     type: 'http',
     scheme: 'bearer',
     bearerFormat: 'JWT',
   },
 };
 
 /**
- * A spec enhancer to add bearer token OpenAPI security entry to
+ * A spec enhancer to add jwt bearer token OpenAPI security entry to
  * `spec.component.securitySchemes`
  */
 @bind(asSpecEnhancer)
 export class SecuritySpecEnhancer implements OASEnhancer {
   name = 'security';
```

Comment:
same as https://github.com/strongloop/loopback-next/pull/4554/files#r376586868, I would rename it to `lb-default-security`